modules/nrfxlib/nrf_802154/sl/platform/nrf_802154_platform_sl_lptimer_zephyr.c

```diff
@@ -79,9 +97,41 @@ static void timer_start_at(struct timer_desc *timer, uint64_t target_time)
 	nrf_802154_sl_mcu_critical_exit(state);
 }
 
+static bool hw_task_state_set(enum hw_task_state_type expected_state,
+			      enum hw_task_state_type new_state)
+{
+	return atomic_cas(&m_hw_task.state, expected_state, new_state);
+}
+
+static void cc_bind_to_ppi(int32_t cc_num, uint32_t ppi_num)
+{
+	uint32_t event_address = z_nrf_rtc_timer_compare_evt_address_get(cc_num);
```

Comment:
Wouldn't it be a good idea to save the event address to a static variable (for example `m_hw_task.cc_event`) in `nrf_802154_platform_sl_lptimer_hw_task_prepare` when allocating the channel?

Then instead of passing `cc_num` to  `cc_unbind` and `cc_bind_to_ppi` we could pass `cc_event`, and there would be no need to call `z_nrf_rtc_timer_compare_evt_address_get` each time.